src/synology_dsm/synology_dsm.py

```diff
@@ -494,3 +496,10 @@ def utilisation(self) -> SynoCoreUtilization:
         if not self._utilisation:
             self._utilisation = SynoCoreUtilization(self)
         return self._utilisation
+
+    @property
+    def audio_station(self) -> SynoAudioStation:

```

Comment:
please move it before ` def download_station(self)`, so the properties are alphabetical ordered